### PR TITLE
Fix minor code issues

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This change fixes minor code issues I found while working on another bug.
| Type?         | refacto
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | I do not really know 🤔

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18425)

### CODE REVIEW

File: classes/pdf/HTMLTemplateDeliverySlip.php [hunk 1]
```diff
@@ -30,6 +30,7 @@
 class HTMLTemplateDeliverySlipCore extends HTMLTemplate
 {
     public $order;
+    public $order_invoice;
```
Comment (Progi1984) [inline classes/pdf/HTMLTemplateDeliverySlip.php:35]: Please add PHPDoc :)

Comment (PululuK) [reply classes/pdf/HTMLTemplateDeliverySlip.php:35]: Hi @Progi1984
This is not the purpose of my PR, in this case I must add PHPDoc for all the attributes ?

Comment (Progi1984) [reply classes/pdf/HTMLTemplateDeliverySlip.php:35]: Mainly for your added attributes ;)

Comment (PululuK) [reply classes/pdf/HTMLTemplateDeliverySlip.php:35]: Done ! Thanks

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
> | How to test?  | I do not really know 🤔

@PululuK no worries, this PR does not need QA 👍

**Review by matks — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @PululuK

**Pull request merged**